### ISSUE TITLE
Configuration warning unrecognized property 'caching'

### ISSUE THREAD

**Issue by EddyArellanes** [DianaIonita/serverless-api-gateway-caching#76]
With the property caching inside like this one:
```
 companies:
    handler: src/index.companiesHandler
    events:
      - http:
          path: '{version}/companies'
          method: ANY
          cors:
            origin: "*"
            headers: ${self:custom.allowed-headers}
            allowCredentials: true
          private: false
          caching:
            enabled: true #false
            ttlInSeconds: 120 #3600
            cacheKeyParameters:
              - name: request.querystring.regionId
              - name: request.querystring.skip
              - name: request.querystring.take
              - name: request.querystring.filterName
              - name: request.querystring.filterValue
```
The Service configuration validation throws  Configuration warning unrecognized property 'caching'.
According to service configuration validation page: 
_Configuration related to external plugin does not have an associated JSON Schema. In such cases, please report the issue with the plugin author and provide them the details on how to extend validation schema in order to permanently correct the issue._

**Comment by DianaIonita** [comment]
Hi @EddyArellanes,

Thanks for raising this issue. Unfortunately, from my initial research, the Serverless framework doesn't currently offer a way to extend the validation schema for `http` events. I have [raised an issue](https://github.com/serverless/serverless/issues/8458) with them, hopefully a solution can be found.

**Comment by yeukfei02** [comment]
@DianaIonita 

I am using serverless v2.11.0
in one of my serverless aws lambda project, I got the same error like this:

`
Serverless:   at 'functions.createPartner.events[0].http': unrecognized property 'caching'
`

wait for the fix/serverless their update...

**Comment by DianaIonita** [comment]
Hi @yeukfei02,

I'm just waiting on [this PR](https://github.com/serverless/serverless/issues/8485#event-3973217677) to get released by the serverless team, then I'll add the validation schema to this plugin.

**Comment by yeukfei02** [comment]
@DianaIonita thats good wait for the update!

**Comment by DianaIonita** [comment]
Released [v1.5.2](https://github.com/DianaIonita/serverless-api-gateway-caching/releases/tag/1.5.2).
